server/s3_put_object_action.cc

```diff
@@ -841,6 +830,119 @@ void S3PutObjectAction::add_object_oid_to_probable_dead_oid_list_failed() {
   s3_log(S3_LOG_DEBUG, "", "%s Exit", __func__);
 }
 
+void S3PutObjectAction::_add_object_oid_to_probable_dead_oid_list(
```

Comment:
I don't see other function names which starts with under score( _ )

Reply:
There is one - S3PutObjectAction::_set_layout_id().